cpp/tests/community/mg_mis_test.cu

```diff
@@ -0,0 +1,255 @@
+        multi_gpu ? src_inclusion_cache.view()
+                  : cugraph::detail::edge_major_property_view_t<vertex_t, vertex_t const*>(
+                      inclusiong_flags.data()),
+        multi_gpu ? dst_inclusion_cache.view()
+                  : cugraph::detail::edge_minor_property_view_t<vertex_t, vertex_t const*>(
+                      inclusiong_flags.data(), vertex_t{0}),
+        cugraph::edge_dummy_property_t{}.view(),
+        [] __device__(auto src, auto dst, auto src_included, auto dst_included, auto wt) {
+          return (src == dst) ? 0 : dst_included;
+        },
+        vertex_t{0},
+        cugraph::reduce_op::plus<vertex_t>{},
+        d_total_outgoing_nbrs_included_mis.begin());
+
+      RAFT_CUDA_TRY(cudaDeviceSynchronize());
+
+      std::vector<vertex_t> h_total_outgoing_nbrs_included_mis(
+        d_total_outgoing_nbrs_included_mis.size());
+      raft::update_host(h_total_outgoing_nbrs_included_mis.data(),
+                        d_total_outgoing_nbrs_included_mis.data(),
+                        d_total_outgoing_nbrs_included_mis.size(),
+                        handle_->get_stream());
+
+      RAFT_CUDA_TRY(cudaDeviceSynchronize());
+
+      {
+        auto vertex_first = mg_graph_view.local_vertex_partition_range_first();
+        auto vertex_last  = mg_graph_view.local_vertex_partition_range_last();
+
+        std::for_each(h_mis.begin(),
+                      h_mis.end(),
+                      [vertex_first, vertex_last, &h_total_outgoing_nbrs_included_mis](vertex_t v) {
+                        ASSERT_TRUE((v >= vertex_first) && (v < vertex_last))
+                          << v << " is not within vertex parition range" << std::endl;
+
+                        ASSERT_TRUE(h_total_outgoing_nbrs_included_mis[v - vertex_first] == 0)
+                          << v << "'s neighbor is included in MIS" << std::endl;
+                      });
+      }
+    }
+  }
+
+ private:
+  static std::unique_ptr<raft::handle_t> handle_;
+};
+
+template <typename input_usecase_t>
+std::unique_ptr<raft::handle_t> Tests_MGMaximalIndependentSet<input_usecase_t>::handle_ = nullptr;
+
+using Tests_MGMaximalIndependentSet_File =
+  Tests_MGMaximalIndependentSet<cugraph::test::File_Usecase>;
+using Tests_MGMaximalIndependentSet_Rmat =
+  Tests_MGMaximalIndependentSet<cugraph::test::Rmat_Usecase>;
+
+TEST_P(Tests_MGMaximalIndependentSet_File, CheckInt32Int32FloatFloat)
+{
+  run_current_test<int32_t, int32_t, float, int>(
+    override_File_Usecase_with_cmd_line_arguments(GetParam()));
+}
+
+TEST_P(Tests_MGMaximalIndependentSet_Rmat, CheckInt32Int32FloatFloat)
+{
+  run_current_test<int32_t, int32_t, float, int>(
+    override_Rmat_Usecase_with_cmd_line_arguments(GetParam()));
+}
+
+TEST_P(Tests_MGMaximalIndependentSet_Rmat, CheckInt32Int64FloatFloat)
+{
+  run_current_test<int32_t, int64_t, float, int>(
+    override_Rmat_Usecase_with_cmd_line_arguments(GetParam()));
+}
+
+INSTANTIATE_TEST_SUITE_P(
+  file_test_pass,
```

Comment:
Why did you name this to `file_test_pass`? We typically name this to `file_test`.